File: exercicios-cpp/estrutura-sequencial/exe05.cpp
#include <iostream>
#include <locale>

/*
Faça um Programa que converta metros para centímetros.
*/

using namespace std;

int main(){
	setlocale(LC_ALL, "portuguese");
	
	float metros=0, centimetros=0;
	
	cout << "Informe a quantidade de metros: "; cin >> metros;
	
	centimetros = metros * 100;
	
	cout << endl; // quebra de linha
	cout << metros << " m em centimetros é " << centimetros << " cm";
	
	
}

File: exercicios-python/estrutura-sequencial/exe05.py
'''
Faça um Programa que converta metros para centímetros.
'''
# 0,01 m = 1 cm -> para converter metros para centimetros é so multiplicar por 100

metros = float(input('Informe a quantidade de metros: '))

conversao = metros * 100

print('{:.2f} m em centimetros é {:.2f} cm'.format(metros, conversao))
